docs/reference/indices/split-index.asciidoc

```diff
@@ -121,11 +121,15 @@ newly split index.
 A split operation:
 
 . Creates a new target index with the same definition as the source
-  index, but with a larger number of primary shards.
+  index, but with a larger number of primary shards. Created shards allocate
+  to the same nodes as their correlating source primary shard, so it must
+  already have sufficient disk to host the copy of the data.
 
 . Hard-links segments from the source index into the target index. (If
   the file system doesn't support hard-linking, then all segments are copied
   into the new index, which is a much more time consuming process.)
++
+TIP: Elastic Cloud's backing file systems do not support hard linking.
```

Comment:
This is probably best answered by @elastic/es-core-infra team who did https://github.com/elastic/elasticsearch/issues/61145 . Adding the author of that PR to this (but anyone from the team feel free to step in to review).

Reply:
Regarding #61145, we did merge a change for it but ended up removing it because, it turned out, Cloud was in fact using a filesystem that correctly reported quotas (XFS IIRC).

Regarding this tip in the docs, I can only imagine it has something to do with attempting to hard-link across Docker volumes, which doesn't work, but you'd need to check with someone from the Hosted ESS team to get an up-to-date answer.

Reply:
@stefnestor might you know who to include from Hosted ESS team to pitch in / review?